build/images/client/Dockerfile

```
@@ -1,13 +1,12 @@
-FROM alpine:3
+FROM alpine:3 as builder
 
 RUN mkdir -p /root/.porter/runtimes && \
     mkdir -p /root/.porter/mixins/exec/runtimes
 
 COPY bin/dev/porter-linux-amd64 /root/.porter/porter
 COPY bin/mixins/exec/dev/exec-linux-amd64 /root/.porter/mixins/exec/exec
 RUN ln -s /root/.porter/porter /root/.porter/runtimes/porter-runtime && \
-    ln -s /root/.porter/mixins/exec/exec /root/.porter/mixins/exec/runtimes/exec-runtime && \
-    ln -s /root/.porter/porter /usr/local/bin/porter
+    ln -s /root/.porter/mixins/exec/exec /root/.porter/mixins/exec/runtimes/exec-runtime
 
```

```suggestion
ENV PATH "$PATH:/root/.porter"

```
Belay removal of this line -- failed to note that the porter mixin install was in the build stage.

---

I'm not installing mixins anymore so that's not a problem anymore.